mmedit/models/backbones/vfi_backbones/flavr_net.py

```diff
@@ -0,0 +1,571 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+from mmcv.runner import load_checkpoint
+
+from mmedit.models.registry import BACKBONES
+from mmedit.utils import get_root_logger
+
+
+@BACKBONES.register_module()
+class FLAVRNet(nn.Module):
+    """PyTorch implementation of FLAVR for video frame interpolation.
+
+    Paper:
+        FLAVR: Flow-Agnostic Video Representations for Fast Frame Interpolation
+    Ref repo: https://github.com/tarun005/FLAVR
+
+    Args:
+        input_frames (int): Number of input frames.
```

Comment:
`num_input_frames`, and `num_output_frames` are better because it is an integer number instead of a list of indices.